app/src/main/java/de/westnordost/streetcomplete/quests/police_type/AddPoliceType.kt

```diff
@@ -0,0 +1,32 @@
+package de.westnordost.streetcomplete.quests.police_type
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.osmquest.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder
+import de.westnordost.streetcomplete.data.quest.NoCountriesExcept
+
+class AddPoliceType : OsmFilterQuestType<PoliceType>() {
+
+    override val elementFilter = """
+        nodes, ways, relations with
```

Comment:
Probably it's only nodes, right?

Reply:
`ways` and `realations` are there because I run a [Overpass query](https://overpass-turbo.eu/s/15p8)* and found out that there are also a lot of areas and relations with `amenity=police` and no `operator`

\* The query is very slow

Reply:
Indeed: 1434 nodes, 1366 ways, 61 relations. Probably `relations` can be dropped then.

Reply:
Will the amenities within those relations be mapped as well?

Reply:
Data consumers expect relations not to have `amenity=police`, because the [OSM wiki](https://wiki.openstreetmap.org/wiki/Tag:amenity=police) says "should not be used on relations" in the infobox at the right (the icon's mouse-hover tooltip text). The [Italian amenity=police](https://wiki.openstreetmap.org/wiki/IT:Tag:amenity=police) article also has the same provision. As a matter of fact, relations are only a thousand worldwide, which is usually not deemed sufficient to implement support.

Relations should be marked with a note (most likely, dedicated nodes and/or areas will be made, but this needs to be handled case by case, as always) or a discussion on lifting the wiki prescription (and on relation-specific conventions or tags to use in combination) should be posed in the [worldwide](https://lists.openstreetmap.org/listinfo/tagging) or [Italian](https://lists.openstreetmap.org/listinfo/talk-it) tagging mailing lists

Reply:
good catch

> Data consumers expect relations not to have amenity=police, because the OSM wiki says "should not be used on relations" in the infobox at the right (the icon's mouse-hover tooltip text)

note that it excludes multipolygons, this is listed as areas in infobox, see https://wiki.openstreetmap.org/wiki/Template:Description#Parameters

I opened https://wiki.openstreetmap.org/wiki/Template_talk:Description#Misleading_tooltip_.28alt_text.29_for_relation

Reply:
Perfect, thanks for the information! I'll remove relations then! 🤗